urns/urns_test.go

```diff
@@ -274,6 +287,13 @@ func TestValidate(t *testing.T) {
 		{"freshchat:+12067799294", "invalid freshchat id"},
 
 		{"slack:U0123ABCDEF", ""},
+
+		// teams must have an 'a:' before the uuid followed by ':serviceURL:' and a respective valid url
```

Comment:
can we get a little more explanation as to what these parts mean?

Reply:
Every conversation id in Teams starts with `a:` and it is with it that we make it explicit where we want to send a message. The `serviceURL` is the endpoint to which we must send the message, Teams use a kind of URL mapping by region, for example, Brazilian accounts have the serviceURL `https://smba.trafficmanager.net/br` like this mapping is not available they always indicate that we receive a message that stores the serviceURL of this particular contact so that future messages can be sent. The urn turned out to be big, but it was a good way to store a contact's serviceURL.

Reply:
Ok then I'm wondering if we need all the information in the URN. For example

```
teams:a:1234-abcd-1234:serviceURL:https://test.example.com
teams:a:3456-bcde-3456:serviceURL:https://smba.trafficmanager.net/br
```

become...

```
teams:1234-abcd-1234:test.example.com
teams:3456-bcde-3456:smba.trafficmanager.net/br
```

Reply:
Removing this information would make it cleaner and I think we can remove them.

Reply:
Tho I'm also wondering if the service URL shouldn't just be a property of the channel. Surely when you add a teams channel don't you need to specify the endpoint like smba.trafficmanager.net/br ?

Reply:
No, this serviceURL property belongs to the contact who will be able to communicate with the Teams channel, so each user who exchanges messages with the channel will have their serviceURL